pandas/core/generic.py

```diff
@@ -1652,6 +1652,62 @@ def _repr_latex_(self):
     strings before writing.
     """
 
+    _shared_docs['to_pickle'] = """
+    Pickle (serialize) %(klass)s object to input file path.
+
+    Parameters
+    ----------
+    path : string
+        File path where the pickled %(klass)s object will be stored.
+    compression : {'infer', 'gzip', 'bz2', 'xz', None}, default 'infer'
+        A string representing the compression to use in the output file.
```

Comment:
Can you add that the default 'infer' infers it from the specified path?

Reply:
added. 💯